conf/declaration.routes

```diff
@@ -496,12 +496,14 @@ GET         /amend-summary                       controllers.declaration.Summary
 
 GET         /saved-summary                       controllers.declaration.SummaryController.displayPage(mode: models.Mode = models.Mode.Draft)
 
-POST        /summary                             controllers.declaration.SummaryController.submitDeclaration()
+POST        /summary                             controllers.declaration.SummaryController.submitDeclaration
 
 # Confirmation
 
-GET         /confirmation                        controllers.declaration.ConfirmationController.displaySubmissionConfirmation()
+GET         /holding                             controllers.declaration.ConfirmationController.displayHoldingConfirmation
 
-GET         /draft-saved                         controllers.declaration.ConfirmationController.displayDraftConfirmation()
+GET         /confirmation                        controllers.declaration.ConfirmationController.displaySubmissionConfirmation
```

Comment:
Polling will be implemented via a future ticket.

Reply:
The AJAX polling script will need its own endpoint (to check for arrived notifications) so maybe the holding page could share the `/confirmation` endpoint to avoid creating two new endpoints perhaps? Leave it up to you though :slightly_smiling_face: